### ISSUE TITLE
Please enable cephfs and rbd kernel modules

### ISSUE THREAD

**Issue by bassam** [docker/for-mac#1781]
### Actual behavior

```
> docker run -it alpine zcat /proc/config.gz | grep CEPH      
# CONFIG_CEPH_LIB is not set
# CONFIG_CEPH_FS is not set

> docker run -it alpine zcat /proc/config.gz | grep RBD 
# CONFIG_BLK_DEV_DRBD is not set
# CONFIG_BLK_DEV_RBD is not set
```
### Information

Can you please enable the rbd and ceph kernel modules in the docker for mac? Setting the 'm' so that they need to be modprobe'd explicitly is OK too.

**Comment by jasonbivins** [comment]
Hi @bassam 
I entered this in as an enhancement request for consideration for future versions.

**Comment by bassam** [comment]
any update on this? Happy to help make it happen.

**Comment by bassam** [comment]
any chance this can be added before DockerCon 2018?

**Comment by pgayvallet** [comment]
Hello,

First, I want to apologize for the (long) delay.

Our VM is based on linuxkit kernel.

Created the issue on their tracker : https://github.com/linuxkit/linuxkit/issues/2960. PR is welcome if you feel capable to contribute.

**Comment by rn** [comment]
FYI, this is now in the upstream LinuxKit kernel via https://github.com/linuxkit/linuxkit/pull/2963 and should trickle down to edge/stable, probably with the next update

**Comment by bassam** [comment]
@rn fantastic! Thanks

**Comment by pgayvallet** [comment]
Will be shipped in 18.04-rc1

**Comment by k4k** [comment]
This was resolved as of 18.04.0-ce-rc2

**Comment by rn** [comment]
Thanks, lost track of when we actually shipped it. Closing
